pkg/cli/proxy_get_test.go

```diff
@@ -22,7 +22,7 @@ func TestIsMeshedPod(t *testing.T) {
 			pod: corev1.Pod{
 				ObjectMeta: metav1.ObjectMeta{
 					Name:   "pod-1",
-					Labels: map[string]string{constants.EnvoyUniqueIDLabelName: "test"},
+					Labels: map[string]string{constants.EnvoyUniqueIDLabelName: "01066be8-c96b-4136-9e93-4c852d4137ea"},
```

Comment:
Addresses https://github.com/openservicemesh/osm/pull/3881#discussion_r679558826

Reply:
Use `uuid.New().String()` here instead of hardcoding it.

Reply:
Already being fixed correctly in https://github.com/openservicemesh/osm/pull/3900. We can drop this change and rebase onto https://github.com/openservicemesh/osm/pull/3900 once merged.

Reply:
Changes dropped.